entrypoint.sh

```diff
@@ -54,8 +54,7 @@ echo "\"buildName\":\"GitHub Actions Run #${INPUT_GITHUB_RUN_ID}\",\"buildOrder\
 mv ./executor.json ./${INPUT_ALLURE_RESULTS}
 
 #environment.properties
-echo "URL=${GITHUB_PAGES_WEBSITE_URL}" >> environment.properties
-mv ./environment.properties ./${INPUT_ALLURE_RESULTS}
+echo "URL=${GITHUB_PAGES_WEBSITE_URL}" >> ./${INPUT_ALLURE_RESULTS}/environment.properties
```

Comment:
Will it work, if we don't have `environment.properties` file in allure-results folder?